lldb/docs/python_extensions.rst

```diff
@@ -0,0 +1,36 @@
+Python Extensions
+=================
+
+LLDB provides many scriptable extensions to augment the debugger capabilities
+and give the ability to the user to tailor their experience to their own needs.
+
+This page describes some of these scripting extension:
+
+Operating System Thread Plugins
+-------------------------------
+
```

Comment:
A short blurb explaining what each of these are for would be useful. It may not be obvious what "Operating System Thread Plugins" are for, as an example.

Reply:
![image](https://github.com/llvm/llvm-project/assets/10001166/0dec76ec-ae53-4ea1-b5f5-b96e64cfb7de)

We get the blurb from the docstrings.

Reply:
Ah, this looks great! Thanks for showing me what it will look like :)